dataframe-geo/build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+import org.jetbrains.kotlin.gradle.tasks.BaseKotlinCompile
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    with(libs.plugins) {
+        alias(kotlin.jvm)
+        alias(publisher)
+        alias(jupyter.api)
```

Comment:
Ok, but comment it because it adds too many linting warnings

Reply:
I mean, there's a reason they show up. Please fix them. Most of the time you can run `gradlew ktLintFormat` and it will auto-fix. For other cases, I recommend installing https://plugins.jetbrains.com/plugin/15057-ktlint which can read the .editorconfig file and highlight exactly what's not linted.